packages/portfolio-deploy/src/axelar-configs.js

```diff
@@ -59,6 +59,8 @@ const AxelarChainIdMap = harden({
  * @typedef {`0x${string}`} HexAddress
  * @typedef {Record<string, HexAddress>} EvmAddressesMap
  * @typedef {{ mainnet: EvmAddressesMap, testnet: EvmAddressesMap }} AddressesMap
+ * @typedef {Record<string, Record<string, HexAddress>>} EvmVaultAddressesMap
+ * @typedef {{ mainnet: EvmVaultAddressesMap, testnet: EvmVaultAddressesMap }} VaultAddressesMap
```

Comment:
treating testnet config as design-time constants (i.e. source code) doesn't appeal to my intuition. I'd rather see it as a thing that a `.build.js` gets from the cli or env or file or net.